packages/vue-components/src/ScrollTopButton.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+  <i
+    id="scroll-top-button"
```

Comment:
I'm a bit unsure about having a fixed ID here - IDs should be unique across a HTML document so if a user defines 2 scroll-to-top buttons (unlikely, but possible) we will have duplicated IDs. Is it possible to avoid this duplication?

We do have some duplicated IDs in our docs as a result of `includes` but it's probably better avoid that :p